app/helpers/feeder_form_builder.rb

```diff
@@ -140,7 +140,10 @@ def add_changed(method, opts)
 
     if object.present?
       if object.respond_to?("#{method}_changed?")
-        add_class(opts, CHANGED_CLASS) if object.public_send("#{method}_changed?")
+        # ignore nils being set to blanks by text fields
+        if !object.respond_to?("#{method}_was") || (object.public_send("#{method}_was").nil? && object.public_send(method) == "")
```

Comment:
@radical-ube We're not sure what line 144 checks for, when on line 145-6`'s `object.public_send("#{method}_changed?")` seems to also be checking for a change in attribute.

Reply:
I was hitting an issue where, if you went to create a new episode, left the title blank, and hit save...

You'd get the angry red "title can't be blank" message on the page.

But you'd also get a ton of orange-outlined changed fields.  Because the form just changed a bunch of attributes (`clean_title`, for instance) from `nil` to `""`.

Bit cosmetic, but I'm leaving off the orange-outline `is-changed` class if all of:

- `object.clean_title_changed? == true`
- `object.clean_title == ""`
- `object.clean_title_was == nil`

Reply:
Oh wait... I see what you're saying.

Yeah, I botched this one.  Will clean this up.